docs/user_guide/mitigation/index.rst

```diff
@@ -0,0 +1,81 @@
+.. _mitigation:
```

Comment:
Any changes here or just copy-paste?

Reply:
Should just be copy-pasting into three separate files.